st3/lsp_utils/__init__.py

```diff
@@ -1,2 +1,12 @@
-from .npm_client_handler import NpmClientHandler
-from .server_npm_resource import ServerNpmResource
+try:
+    from .v1 import ApiWrapper
+    from .v1 import NpmClientHandler
+except ImportError:
+    from .v0 import ApiWrapper  # type: ignore
+    from .v0 import NpmClientHandler
```

Comment:
I think I'd prefer an explicit variant that does selection based on the version of LSP.
I can imagine in the future there might be some API changes that don't cause the import to fail and that approach would not work then.

Reply:
On the other hand I don't imagine us doing such drastic API changes that we would need to create a separate file for them...